backend/src/main/java/ddangkong/domain/balance/room/RoomContent.java

```diff
@@ -35,6 +38,24 @@ public class RoomContent extends BaseEntity {
     @Column(nullable = false)
     private int round;
 
+    private LocalDateTime roundEndedAt;
+
+    @Column(nullable = false)
+    private boolean isUsed;
+
+    public static List<RoomContent> createList(Room room, List<BalanceContent> balanceContents) {
```

Comment:
의견) 
```suggestion
    public static List<RoomContent> createRoomContents(Room room, List<BalanceContent> balanceContents) {
```
타입은 언제든지 변경될 수 있으니 복수형으로 나타내는 것이 좋을 것 같아요